README.md

```diff
@@ -2,10 +2,27 @@
 
 Alpha version of the ROS2 Universal Robots driver. Should be transferred to the Universal Robots org when ready.
 
+## Requirements
+
+Follow the [instructions](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver#setting-up-a-ur-robot-for-ur_robot_driver) in the paragraph 
+[`Prepare the robot` ](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver#prepare-the-robot)
+
 ## Build Instructions
 
-To build this package in a ROS Foxy workspace, clone this repo
-into the `src/` directory, then:
+To build this package follow the [instructions](https://index.ros.org/doc/ros2/Installation/Foxy/Linux-Install-Debians/) for installation of ROS2.
+
+After installation create a ROS Foxy workspace:
```

Comment:
Maybe better link [official docs](https://index.ros.org/doc/ros2/Tutorials/Workspace/Creating-A-Workspace/) for this general stuff

Reply:
https://github.com/PickNikRobotics/Universal_Robots_ROS2_Driver/pull/33/commits/e1b1db0928450f67e7159aa3f4dbc79004015795